app/views/tasks/show.html.erb

```diff
@@ -0,0 +1,14 @@
+<h2 class = "showheader"> Here is info about this task:</h2>
+
+<% if @task.nil? %>
+  <h1> 404 Not found</h1>
+<% else %>
+
+
+    <h3 class = "task_name">  <%= @task.name %> </h3>
+
+  <h4>Description: <%= @task.description%></h4>
+
+  <%= link_to "Delete #{@task.name} |", destroy_task_path(@task.id),class: "link_to", method: :delete, data: {confirm: 'Are you sure?'} %>
```

Comment:
You should probably have a link to mark a task complete/incomplete on the show action.